src/services/item/plugins/app/chatBot/schemas.ts

```diff
@@ -0,0 +1,26 @@
+import { ChatbotRole } from '@graasp/sdk';
+
+const chatBotDefinition = {
+  type: 'object',
+  required: ['role', 'content'],
+  properties: {
+    role: { type: 'string', enum: Object.values(ChatbotRole) },
+    content: { type: 'string' },
+  },
+};
+
+const create = {
+  body: {
```

Comment:
It seems you are missing the schema validation for the query string:
Here is an example:
```ts
querystring: {
    type: 'object',
    properties: {
      gptVersion: { type: 'string', enum: Object.values(GTPVersion) },
    },
    additionalProperties: false,
  },
```